test-chart/action.yaml

```
@@ -44,8 +44,12 @@ runs:
         BIN_DIR=$HOME/bin
         mkdir -p $BIN_DIR
         VERSION=${{ steps.get-version.outputs.version }}
-        curl -sLo $BIN_DIR/testchart.tar.gz https://github.com/silphid/testchart/releases/download/v$VERSION/testchart_${VERSION}_linux_amd64.tar.gz
-        tar xvzf $BIN_DIR/testchart.tar.gz -C $BIN_DIR
+        curl \
+          --header "Authorization: Bearer ${{ github.token }}" \
+          --location \
+          output - \
```

This looks weird to me. It should be `--output`, right?


```suggestion
          --output - \
```

---

Yep